contracts/SpokePool.sol

```diff
@@ -1535,13 +1574,9 @@ abstract contract SpokePool is
         }
     }
 
-    // Determine whether the combination of exlcusiveRelayer and exclusivityDeadline implies active exclusivity.
-    function _fillIsExclusive(
-        address exclusiveRelayer,
-        uint32 exclusivityDeadline,
-        uint32 currentTime
-    ) internal pure returns (bool) {
-        return exclusivityDeadline >= currentTime && exclusiveRelayer != address(0);
+    // Determine whether the exclusivityDeadline implies active exclusivity.
+    function _fillIsExclusive(uint32 exclusivityDeadline, uint32 currentTime) internal pure returns (bool) {
```

Comment:
@pxrl wdyt of this:

I don't think there's a need to check that the `exclusiveRelayer` is not the zero address. The `_depositV3` function protects depositors from accidentally setting an exclusivity period but not setting an exclusive relayer.

If a filler tries to call fillV3 with an exclusivity period but no exclusive relayer, then they won't be able to fill the deposit. This is also intuitive because this type of deposit would no longer be possible.

In fact, I don't think we needed to check this `exclusiveRelayer != address(0)` before, but I could be missing something

Reply:
+1 from me! I was keen for that in a previous change: https://github.com/across-protocol/contracts/pull/584#discussion_r1725757479